__tests__/offlineUI.js

```diff
@@ -209,9 +209,93 @@ describe('offline UI', async () => {
 
       })
     });
+    // Snapshots are intentionally not compared because the spinner will cause the snapshot to fail regularly
+    // Uncomment the following lines to re-capture snapshot or do a manual comparison
 
-    const base64PNG = await driver.takeScreenshot();
+    // const base64PNG = await driver.takeScreenshot();
```

Comment:
Because of the gif in ConnectivityStatus, these screenshots will regularly fail. A previous suggestion was to take the screenshot and not do a comparison, and because there's been a lot of feedback I've been playing with what to comment out, if anything, so manual tests can still be done.

I got a new suggestion yesterday where I will temporarily replace the gif with a png during the tests so they won't fail regularly. This also allows customers to insert their own spinner if they want to via `defaultStyleSetOptions`. I'll make that a part of this PR (and the commented code will be gone).